### PR TITLE
Automatic slack deploy notifications

### PR DESCRIPTION
**Pull request by jrochkind**

### CODE REVIEW

File: lib/chf/slackistrano_messaging.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module Chf
+  class SlackistranoMessaging < Slackistrano::Messaging::Base
+    def channels_for(action)
+      channels = if fetch(:stage).to_s == "production"
+        ["#digital-general"]
+      else
+        ["#{}digital-technical"]
```
Comment (hackmastera) [inline lib/chf/slackistrano_messaging.rb]: What do these braces mean? It seems like that would negate the presence of the `#`.

Comment (jrochkind) [reply lib/chf/slackistrano_messaging.rb]: bug/typo, yup. Wonder how it was working anyway? Will remove.

Comment (hackmastera) [reply lib/chf/slackistrano_messaging.rb]: Editor being too clever?

**Comment by jrochkind** [comment]
yup it was editor being too clever.

**Pull request merged**